### ISSUE TITLE
Client doesn't validate certificates

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#13]
```
The httplib2 library that google-api-python-client relies upon does not do 
certificate validation. For potential solutions see:


  fancy_urllib and http://selenic.com/hg/file/tip/mercurial/url.py#l509


```

Original issue reported on code.google.com by `jcgregorio@google.com` on 15 Mar 2011 at 3:00

**Comment by GoogleCodeExporter** [comment]
Original comment by `jcgregorio@google.com` on 15 Mar 2011 at 3:00
- Changed title: **Client doesn't validate certificates**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with sync to version 0.7.0 of httplib2: 
http://code.google.com/p/google-api-python-client/source/detail?r=3ce2ae291c18f2
2704e53783150d9b4621a6d055
```

Original comment by `jcgregorio@google.com` on 14 Jun 2011 at 6:33
- Changed state: **Fixed**
